rust/vectordb/src/index/vector.rs

```diff
@@ -24,6 +24,7 @@ pub trait VectorIndexBuilder {
     fn get_replace(&self) -> bool;
 }
 
+#[derive(Default)]
```

Comment:
I think the default of `bool` is `false` we might want to manually implement default here

Reply:
Fixed